src/clj/runbld/notifications/slack.clj

```diff
@@ -8,15 +8,18 @@
             [runbld.notifications :as n]
             [runbld.store :as store]
             [stencil.core :as mustache]
-            [elasticsearch.document :as doc]))
+            [elasticsearch.document :as doc]
+            [robert.bruce :refer [try-try-again] :as try]))
 
 (defn api-send
   "Make the Slack REST API call"
   [opts js hook]
   (do
     ((opts :logger) "NOTIFYING SLACK")
-    (http/post hook
-               {:body js})))
+    (try-try-again
+     {:sleep 250 :tries 10 :decay try/exponential}
```

Comment:
This might be a bit excessive. Ends up being about 20 min. :1st_place_medal:

Something more along the lines of `:sleep 500 :tries 20 :decay try/linear` to get 10 secs is probably more realistic.